README.rst

```diff
@@ -31,6 +32,12 @@ Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 Changes
 =======
 
+v0.3.3
+------
+
+* ``i18n_tool detect`` command added. This command checks if the GitHub English
```

Comment:
`This command determines if the source translation files are up-to-date. If they are not it returns a non-zero exit code.`

This tool is not dependent on GitHub, and English files don't necessarily have to be the source.